todo-backend/README.adoc

```diff
@@ -498,6 +617,8 @@ $ helm show readme jboss-eap/eap-xp3
 ----
 endif::[]
 
+// Testing on Openshift
+include::../shared-doc/run-arquillian-integration-tests-with-openshift.adoc[leveloffset=+2]
```

Comment:
This shared doc section uses {artifactId} to construct a `mvn clean verify -Parq-remote` command. As a result we get:

`mvn clean verify -Parq-remote -Dserver.host=https://$(oc get route todo-backend --template='{{ .spec.host }}')`

but it must be 

`mvn clean verify -Parq-remote -Dserver.host=https://$(oc get route todo --template='{{ .spec.host }}')`

Reply:
We can postpone this until the https://github.com/wildfly/quickstart/pull/614#issuecomment-1313504461 is clear